_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
+packages that already had to be rebuilt twice a year due to LibreSSL ABI
+breaks, but if anything in your setup depends explicitly on LibreSSL (you know
+who you are!), be sure to migrate away from that.
+
+The important parts of the post are done, but you can read on to learn about
+the decision process and some things we had to do.
+
+The discussion started in an
+[RFC](https://github.com/void-linux/void-packages/issues/20935) from April
+12th, 2020, and has been ongoing since then, with most maintainers expressing
+support for the idea. The main reasons for the switch are as listed in the RFC:
+
+- Less need for patching packaged software, which is made especially difficult
+  and potentially risky by the complexity of the OpenSSL API.
+- Better platform support for accelerated implementation on non `x86_64`
```

Comment:
This point reads weird, I think your qualifiers are in the wrong order.

Reply:
How about "Support for hardware accelerated hashing (? or is it encryption, i dont remember) on non-x86 architectures"

Reply:
"Better hardware acceleration on non-`x86-64` platforms"

Reply:
Sounds nebolous, just say, that we had very poor performance when porting to ppc.

Reply:
I went with `Extensive support for platform specific optimizations outside of x86.`